e2e/config.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     addHearingBookingDetails: 'Add hearing details',
     createNoticeOfProceedings: 'Create notice of proceedings',
     addStatementOfService: 'Add statement of service (c9)',
+    draftStandardDirections: 'Draft standard directions',
```

Comment:
still awaiting input. They wanted issue standard directions order but this was 1 character over the limit I believe

Reply:
Sent a message to Mia

Reply:
An issue has been raised with CCD to extend the character length of event labels. I imagine this will move the correct event title out of scope of this ticket